### ISSUE TITLE
Don't create regions when they already exists (even if they don't match the location you want)

### ISSUE THREAD

**Issue by weha** [BartPermentier/al-toolbox#107]
If regions already exist for a function, they should not be automatically created again.

```
codeunit 123456 "ESCATest"
{
    Access = Public;
    SingleInstance = true;

    #region TestFunction
    [IntegrationEvent(false, false)]
    internal procedure TestFunction()
    begin
    end;
    #endregion TestFunction
}
```

Then execute create regions will create extra regions.
The location for either is probably debatable but creating secondary regions should not happen.

```
codeunit 123456 "ESCATest"
{
    Access = Public;
    SingleInstance = true;

    #region TestFunction
    [IntegrationEvent(false, false)]
    #region TestFunction
    internal procedure TestFunction()
    begin
    end;
    #endregion TestFunction
    #endregion TestFunction
}
```

**Comment by fvet** [comment]
![Regions2](https://user-images.githubusercontent.com/25268332/233649212-8d386639-202e-4325-a351-3bb247f7bc51.gif)

**Comment by dsaveyn** [comment]
We are testing a fix for this internally as we speak.

**Comment by dsaveyn** [comment]
The fix for this will be included in the upcoming 2.0.29 release.
